src/routes/_components/status/Status.html

```diff
@@ -260,7 +260,7 @@
         notification && notification.status &&
           notification.type !== 'mention' && notification.status.id === originalStatusId
       ),
-      spoilerShown: ({ $spoilersShown, uuid }) => !!$spoilersShown[uuid],
+      spoilerShown: ({ $spoilersShown, uuid, $showAllSpoilers }) => ($spoilersShown[uuid] === undefined ? !!$showAllSpoilers : !!$spoilersShown[uuid]),
```

Comment:
```suggestion
      spoilerShown: ({ $spoilersShown, uuid, $showAllSpoilers }) => (typeof $spoilersShown[uuid] === 'undefined' ? !!$showAllSpoilers : !!$spoilersShown[uuid]),
```

Just to match the code style I use elsewhere in the codebase